clients/application-kafka/src/test/java/org/eclipse/hono/application/client/kafka/impl/KafkaBasedCommandSenderTest.java

```diff
@@ -126,4 +148,65 @@ public void testSendOneWayCommandSucceeds(final VertxTestContext ctx) {
                     ctx.completeNow();
                 }));
     }
+
+    /**
+     * Verifies that a command sent to a device succeeds and also a response is received from the device.
+     *
+     * @param ctx The vert.x test context.
+     */
+    @Test
+    public void testSendCommandAndReceiveResponse(final VertxTestContext ctx) {
```

Comment:
@kaniyan This test seems to always fail when run as part of the nightly jenkins job (see e.g. [here](https://ci.eclipse.org/hono/job/Hono-Nightly-Declarative-Pipeline/966/console)).
````
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 5.227 s <<< FAILURE! - in org.eclipse.hono.application.client.kafka.impl.KafkaBasedCommandSenderTest
[ERROR] testSendCommandAndReceiveResponse{VertxTestContext}  Time elapsed: 5.209 s  <<< ERROR!
java.util.concurrent.TimeoutException: The test execution timed out. Make sure your asynchronous code includes calls to either VertxTestContext#completeNow(), VertxTestContext#failNow() or Checkpoint#flag()
````
Can you have a look?